xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -96,62 +95,55 @@
 
 
 class ClockPlacer(object):
-    def __init__(self, conn, io_locs, blif_data):
-        c = conn.cursor()
+    def __init__(self, cmt_dict, io_locs, blif_data, roi):
+        def get_cmt(cmt_dict, loc):
```

Comment:
This appears to be a free function, so please more it to the root?

Reply:
Done